dask/dataframe/optimize.py

```diff
@@ -26,5 +31,9 @@ def optimize(dsk, keys, **kwargs):
     except ImportError:
         dsk4 = dsk2
     dsk5 = fuse_getitem(dsk4, dataframe_from_ctable, 3)
-    dsk6, _ = cull(dsk5, keys)
-    return dsk6
+    if read_parquet_row_group:
+        dsk6 = fuse_getitem(dsk5, read_parquet_row_group, 4)
```

Comment:
Yes, `fuse_getitem` looks for patterns like `(getitem, (somefunc, ..., X), Y)` and fuses them into `(somefunc, ..., Z)` for some `z = f(x, y)`.  It is used in cases like these.

The `fuse` operation is somewhat different.  The docs here aren't bad: http://dask.pydata.org/en/latest/optimize.html